Lib/venv/scripts/common/Activate.ps1

```diff
@@ -48,4 +53,4 @@ if (Test-Path env:PYTHONHOME) {
 
 # Add the venv to the PATH
 copy-item env:PATH env:_OLD_VIRTUAL_PATH
-$env:PATH = "$env:VIRTUAL_ENV\__VENV_BIN_NAME__;$env:PATH"
+$env:PATH = add-bin $env:PATH
```

Comment:
Another way you can do this, without having to create a function, could be like so:

```powershell
$env:PATH = "$(Join-Path -Path $env:VIRTUAL_ENV -ChildPath '__VENV_BIN_NAME__');$env:PATH"
```

...but having said that, I prefer the way you've done it here.